From c9c6d70032404edbf98aae4bf2941a7791bec587 Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 13 Oct 2014 22:30:15 -0400
Subject: [misc] genwqe: Remove unnecessary include

Message-id: <1413239416-7778-28-git-send-email-gduarte@redhat.com>
Patchwork-id: 97549
O-Subject: [RHEL7.1 PATCH BZ 1088491 27/28 v2] GenWQE: Remove unnecessary include
Bugzilla: 1088491
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088491
Upstream Status: 5b35b20d4eb4c7fa6263939304c69207f791535f

commit 5b35b20d4eb4c7fa6263939304c69207f791535f
Author: Kleber Sacilotto de Souza <klebers@linux.vnet.ibm.com>
Date:   Thu Jul 10 10:17:08 2014 -0300

    GenWQE: Remove unnecessary include

    The include for the UAPI header file from card_base.c can be removed
    since it's already included on card_base.h.

    Signed-off-by: Kleber Sacilotto de Souza <klebers@linux.vnet.ibm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/misc/genwqe/card_base.c b/drivers/misc/genwqe/card_base.c
index abb7961..dc8c04a 100644
--- a/drivers/misc/genwqe/card_base.c
+++ b/drivers/misc/genwqe/card_base.c
@@ -38,7 +38,6 @@
 #include <linux/notifier.h>
 #include <linux/device.h>
 #include <linux/log2.h>
-#include <linux/genwqe/genwqe_card.h>
 
 #include "card_base.h"
 #include "card_ddcb.h"
-- 
1.7.1